drivers/iio/light/apds9960.c

```diff
@@ -273,6 +277,15 @@ static const struct iio_event_spec apds9960_als_event_spec[] = {
 
 static const unsigned long apds9960_scan_masks[] = {0xf, 0};
 
+static const struct iio_chan_spec_ext_info adps9960_ext_info[] = {
+	{
+		.name = "near_level",
```

Comment:
I assume this should be `nearlevel` and not `near_level`? At least according to `Documentation/ABI/testing/sysfs-bus-iio-proximity` and the only other driver that provides such a thing.